modules/swagger-models/src/test/java/io/swagger/models/properties/PropertyBuilderTest.java

```diff
@@ -1,26 +1,59 @@
 package io.swagger.models.properties;
 
-import io.swagger.models.properties.Property;
-
 import org.testng.Assert;
-import org.testng.annotations.Test;
-
-import io.swagger.models.properties.StringProperty;
-import io.swagger.models.properties.PropertyBuilder;
 
+import org.testng.annotations.Test;
 
 public class PropertyBuilderTest {
 
     @Test
-    public void testStringFormat() {
-        for (StringProperty.Format format : StringProperty.Format.values()) {
+    public void testStringPredefinedFormats() {
+        for (final StringProperty.Format format : StringProperty.Format.values()) {
             final String name = format.getName();
             final Property property = PropertyBuilder.build(StringProperty.TYPE, name, null);
             Assert.assertEquals(property.getType(), StringProperty.TYPE);
             Assert.assertEquals(property.getFormat(), name);
         }
+    }
+
+    @Test
+    public void testStringNoFormat() {
         final Property noFormat = PropertyBuilder.build(StringProperty.TYPE, null, null);
         Assert.assertEquals(noFormat.getType(), StringProperty.TYPE);
         Assert.assertNull(noFormat.getFormat());
     }
+
+    @Test
+    public void testStringCustomFormat() {
+        final String customFormatName = "custom";
+        final Property customFormatProperty = PropertyBuilder.build(StringProperty.TYPE, customFormatName, null);
+        Assert.assertNotNull(customFormatProperty);
+        Assert.assertEquals(customFormatProperty.getType(), StringProperty.TYPE);
+        Assert.assertEquals(customFormatProperty.getFormat(), customFormatName);
+    }
+
+    @Test
+    public void testNumberFloat() {
+        final String floatFormat = "float";
+        final Property floatProperty = PropertyBuilder.build(DecimalProperty.TYPE, floatFormat, null);
+        Assert.assertEquals(floatProperty.getType(), DecimalProperty.TYPE);
+        Assert.assertEquals(floatProperty.getFormat(), floatFormat);
+    }
```

Comment:
I looked into parameterized Tests with TestNG, and implemented both as parameterized test. Now we have 40 tests (by 4 test methods).

I'll see if I can also cover some corner cases to increase the coverage (currently 67% for PropertyBuilder).